corgi/tasks/manifest.py

```diff
@@ -17,6 +19,8 @@
     retry_kwargs=RETRY_KWARGS,
 )
 def update_manifests():
+    # Ensure the OUTPUT_FILES_DIR exists on first run
+    Path(settings.OUTPUT_FILES_DIR).mkdir(exist_ok=True)
```

Comment:
alternately things like directory creation might be an activity to consider for component-registry-ops ansible ... otherwise LGTM

Reply:
Noted.